sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+	blob_tier:~azure.storage.blob.models.StandardBlobTier
+	blob_tier_change_time:datetime
+	blob_tier_inferred:bool
+	deleted:bool
+	deleted_time:datetime
+	remaining_retention_days:int
+	creation_time:datetime
+	archive_status:str
+
+	def __init__(self, *, name, container, snapshot, blob_type, metadata, last_modified, etag, size, content_range, append_blob_committed_block_count, page_blob_sequence_number, server_encrypted, copy, content_settings, lease, blob_tier, blob_tier_change_time, blob_tier_inferred, deleted, deleted_time, remaining_retention_days, creation_time, archive_status, **kwargs): ...
+	def __str__(self): ...
+	def get(self, key, default=None): ...
+	def has_key(self, k): ...
+	def items(self): ...
+	def keys(self): ...
+	def update(self, *args, **kwargs): ...
+	def values(self): ...
+
+class BlobPropertiesPaged(AsyncPageIterator, AsyncIterator, AsyncIterable, Generic, object):
+	service_endpoint:str
+	prefix:str
+	marker:str
+	results_per_page:int
+	location_mode:str
+	current_page
+	container:str
+	delimiter:str
+	command:callable
+	continuation_token:str
+
+	def __aiter__(self): ...
+	def __anext__(self): ...
+	def __class_getitem__(cls, params): ...
+	def __init__(self, command: callable, container: str=None, prefix: str=None, results_per_page: int=None, continuation_token: str=None, delimiter: str=None, location_mode: str=None, *, service_endpoint, marker, current_page, get_next, extract_data): ...
+	def __init_subclass__(cls, *args, **kwargs): ...
+	def __new__(cls, *args, **kwargs): ...
+
+class BlobServiceClient(AsyncStorageAccountHostsMixin, StorageAccountHostsMixin, object):
+	url:str
+	primary_endpoint:str
+	primary_hostname:str
+	secondary_endpoint:str
+	secondary_hostname:str
+	location_mode:str
+	account_url:str
+	credential
+
+	def __aenter__(self): ...
+	def __aexit__(self, *args): ...
+	def __enter__(self): ...
+	def __exit__(self, *args): ...
+	def __init__(self, account_url: str, credential=None, loop=None, *, url, primary_endpoint, primary_hostname, secondary_endpoint, secondary_hostname, location_mode, **kwargs): ...
+	def create_container(self, name: str, metadata: metadata=None, public_access: public_access=None, timeout: int=None, **kwargs) -> ~azure.storage.blob.aio.container_client_async.ContainerClient: ...
+	def delete_container(self, container: container, lease: ~azure.storage.blob.lease.LeaseClient=None, timeout: int=None, *, if_modified_since, if_unmodified_since, if_match, if_none_match, **kwargs) -> None: ...
+	@classmethod
+	def from_connection_string(cls, conn_str: str, credential=None, **kwargs): ...
+	def generate_shared_access_signature(self, resource_types: resource_types, permission: permission, expiry: expiry, start: start=None, ip: str=None, protocol: str=None) -> str: ...
+	def get_account_information(self, **kwargs) -> dict(str, str): ...
+	def get_blob_client(self, container: container, blob: blob, snapshot: snapshot=None) -> ~azure.storage.blob.aio.blob_client_async.BlobClient: ...
```

Comment:
this should be changed to accept only string for container (which should be renamed to container_name).